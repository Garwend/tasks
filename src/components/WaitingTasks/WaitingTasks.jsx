import Task from "../Task/Task";

const WaitingTasks = ({ waitingTasks, setWaitingTasks }) => {
    return (
        <article>
            <header>Zadania w kolejce</header>
            <ul>
                {waitingTasks.map(task => <Task key={task.id} {...task} setWaitingTasks={setWaitingTasks} />)}
            </ul>
        </article>
    )
}

export default WaitingTasks;