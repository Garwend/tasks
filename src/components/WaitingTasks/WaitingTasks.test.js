import renderer from 'react-test-renderer';
import WaitingTasks from './WaitingTasks';

test('should render WaitingTasks component', () => {
    const tasks = [{id:1, priority: 3, seconds: 5}]
    const tree = renderer.create(<WaitingTasks waitingTasks={tasks} />).toJSON();
    expect(tree).toMatchSnapshot();
})