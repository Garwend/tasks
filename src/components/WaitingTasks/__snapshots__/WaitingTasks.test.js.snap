// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render WaitingTasks component 1`] = `
<article
  className="tasks-wrap"
>
  <header>
    Zadania w kolejce
  </header>
  <ul>
    <li
      className="task"
    >
      <p>
        <span>
          czas: 
        </span>
         
        5
         sekund 
      </p>
      <p>
        <span>
          priorytet: 
        </span>
         
        3
      </p>
      <button
        className="delete-btn"
        onClick={[Function]}
      >
        usu≈Ñ
      </button>
    </li>
  </ul>
</article>
`;
